sr_demos/scripts/sr_demos/demo.py

```diff
@@ -352,8 +379,13 @@ def stored_states_sequence(self):
 
         self.commander.run_named_trajectory(trajectory)
         rospy.loginfo("Stored States demo completed")
+        self._enable_touch_detection(True)
 
     def standard_demo_sequence(self):
```

Comment:
No type hinting?

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/823/commits/4d899740e14c6a37b20146403ea06d3b82a837c8